hack/push.sh

```diff
@@ -8,12 +8,16 @@
 #     .\_/.
 #
 
-
 PACKAGE="$1"
 VERSION="$2"
 REPO="$3"
 
+# Push version tag
 docker tag "${PACKAGE}":"${VERSION}" "${REPO}":"${VERSION}"
 docker push "${REPO}":"${VERSION}"
 
-sleep 10
\ No newline at end of file
+# Push latest tag
+docker tag "${PACKAGE}":"${VERSION}" "${REPO}":latest
+docker push "${REPO}":latest
```

Comment:
the `:latest` tag could serve well for those who want to be on the cutting edge always.

`:latest` does not mean unstable: Everything that is tagged is at least made sure to run and pass end-to-end tests.